packages/plans-grid/src/plans-grid/index.tsx

```diff
@@ -63,15 +65,37 @@ const PlansGrid: React.FunctionComponent< Props > = ( {
 	customTagLines,
 	hidePlansComparison = false,
 	defaultAllPlansExpanded = false,
+	onBillingPeriodChange,
 } ) => {
 	const { __ } = useI18n();
 
+	const [ billingPeriod, setBillingPeriod ] = React.useState< Plans.PlanBillingPeriod >(
+		'ANNUALLY'
+	);
+
+	React.useEffect( () => {
+		if ( onBillingPeriodChange ) {
+			onBillingPeriodChange( billingPeriod );
+		}
+	}, [ billingPeriod, onBillingPeriodChange ] );
+
+	const [ maxMonhtlyDiscountPercentage, setMaxMonhtlyDiscountPercentage ] = React.useState<
+		number | undefined
+	>( undefined );
```

Comment:
Can't we use [the hook](useSupportedPlans) instead of this?

Reply:
The link to the hook is broken for me, can you double check?

Reply:
Sure! I mean this [hook ](https://github.com/Automattic/wp-calypso/pull/48963/files#diff-3242762916fde7013a4cb5ab3f61bfa25b101956ed6b517e9a2ee92b8b009e52R12).

Reply:
Thank you! Done in https://github.com/Automattic/wp-calypso/pull/48963/commits/e2bff7c156d04a06b3983da50e2e638a11fb7e79